docs/stdlib/json.rst

```diff
@@ -272,7 +268,7 @@ possible to cast a JSON value directly into a :eql:type:`tuple`.
 
     :index: json parse loads
 
-    Return JSON value represented by the input *string*.
+    Returns the value of the inputted :eql:type:`str` from a :eql:type:`json`:
```

Comment:
```suggestion
    Returns a :eql:type:`json` value parsed from the given :eql:type:`str`.
    
    Examples:
```
- Rewrite to clarify what goes in a what comes out
- This is a rare case where the types are important enough for the one-liner since taking in one type and spitting out another is the point of the function we're describing. Good call on that.
- Added "Examples" to get rid of the colon in the one-liner